Gemfile

```diff
@@ -42,7 +42,8 @@ gem 'slim-rails'
 gem 'split', require: 'split/dashboard'
 gem 'staccato'
 gem 'twilio-ruby'
-gem 'two_factor_authentication', github: 'Houdini/two_factor_authentication'
+gem 'two_factor_authentication', github: 'amoose/two_factor_authentication',
+                                 branch: 'feature/more_options'
```

Comment:
OK.. can you add a comment with a link to that bug?    This is where I realy like TODO. e.g:

```
TODO(amos): Unfork this gem: https://github.com/18F/identity-private/issues/708
```